src/sentry/static/sentry/app/views/performance/compare/utils.tsx

```diff
@@ -318,17 +323,62 @@ function createChildPairs({
   };
 }
 
+function jaroSimilarity(thisString: string, otherString: string): number {
+  // based on https://winkjs.org/wink-distance/string-jaro-winkler.js.html
+  // and https://en.wikipedia.org/wiki/Jaro%E2%80%93Winkler_distance
+  if (thisString === otherString) {
+    return 1;
+  }
+
+  let jaroDistance: number = jaro(thisString, otherString).distance;
+
+  const scalingFactor = 0.1;
+  const boostThreshold = 0.3;
+
+  if (jaroDistance > boostThreshold) {
+    return 1 - jaroDistance;
+  }
+
+  const pLimit = Math.min(thisString.length, otherString.length, 4);
+  let l = 0;
+
+  for (let i = 0; i < pLimit; i += 1) {
+    if (thisString[i] === otherString[i]) {
+      l += 1;
+    } else {
+      break;
+    }
+  }
+
+  jaroDistance -= l * scalingFactor * jaroDistance;
+  return 1 - jaroDistance;
+}
+
 function matchableSpans({
   baselineSpan,
   regressionSpan,
 }: {
   baselineSpan: SpanType;
   regressionSpan: SpanType;
-}): boolean {
+}): number {
   const opNamesEqual = baselineSpan.op === regressionSpan.op;
-  const descriptionsEqual = baselineSpan.description === regressionSpan.description;
 
-  return opNamesEqual && descriptionsEqual;
+  if (!opNamesEqual) {
+    return 0;
+  }
+
+  // remove whitespace and convert string to lower case as the individual characters
+  // adds noise to the edit distance function
+  const baselineDescription = (baselineSpan.description || '')
```

Comment:
Any other characters we need to consider here?

Reply:
What characters do you have in mind?

Reply:
Good question. I’m not quite sure lol. I think it’s important to keep this mind though. As we do more testing

Reply:
This should be a good start, the Jaro–Winkler  function should be able to do most of the heavy lifting 😄